docs/examples/all-ligand-explicit.rst

```diff
@@ -0,0 +1,197 @@
+.. _all-ligand-explicit:
+
+Absolute Binding of Binders and Non-binders to T4 lysozyme L99A in Explicit Solvent with Combinatorial Options
+==============================================================================================================
+
+This example performs absolute binding free energy calculations for a series of small molecules that contain known
+binders and nonbinders to the T4 Lysozyme L99A protein.
+
+We take advantage of three advanced features of YANK in this example (please see the
+:doc:`basic tutorial for the YAML sections and binding free energy calculation <p-xylene-explicit>`
+for learning how to set up basic YANK simulations):
+
+1. YANK's ability to build out small molecules from SMILES strings (through OpenEye Toolkit)
+
+   * Please see the `OpenEye Install Instructions <http://docs.eyesopen.com/toolkits/python/quickstart-python/install.html>`_ for installing these tool kits
```

Comment:
We have some instructions we should possibly link to:
http://getyank.org/latest/installation.html#optional-tools